src/tv2-common/onTimelineGenerate.ts

```diff
@@ -56,6 +56,7 @@ export interface PieceMetaData {
 	mediaPlayerSessions?: string[]
 	mediaPlayerOptional?: boolean
 	modifiedByAction?: boolean
+	belongsToRemotePart?: boolean // used to find last idents in Lives
```

Comment:
We should strive to make our code not require additional comments.

Would
```
export interface PieceMetaData {
  ...,
  identMetaData?: IdentMetaData
}

interface IdentMetatData {
  belongsToRemotePart: boolean
}
```
make it clear that it's used for idents?

(Maybe it should be `LiveMetaData`? Can't really see how this attribute is used and your comment indicates that the attribute is used on the Live. But the attribute name suggest that it is used on the Ident)